src/Microsoft.Azure.WebJobs.Host/Bindings/OpenType.cs

```diff
@@ -6,28 +6,328 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Bindings
 {
+    // Ensures that open Types match consistently 
+    public class OpenTypeMatchContext
```

Comment:
Should this be public?

Reply:
Unfortunately, it needs to be since public abstract OpenType.IsMatch(Type type, OpenTypeMatchContext context); needs to be public. 
That method needs to be public since derived classes may implement it. But the only thing a derived class would do with this context is pass it into a IsMatch call on one of our builtin opentypes. 

---
In reply to: [148666948](https://github.com/Azure/azure-webjobs-sdk/pull/1413#discussion_r148666948) [](ancestors = 148666948)